doc/rtd/reference/custom_modules/custom_mergers.rst

```diff
@@ -0,0 +1,66 @@
+.. _custom_mergers:
```

Comment:
This section doesn't address how to use or where to place a custom merger. Can merge syntax be specified in user data for a custom merger like it can for a built in merger?

I know you didn't write this page and are just moving the old sections here, so I don't think that these issues need to be addressed in this PR, but we should probably file a follow-up ticket to address them.

Reply:
https://github.com/canonical/cloud-init/issues/5560